.goreleaser.yaml

```diff
@@ -121,3 +124,18 @@ release:
 
   footer: |
     **Full Changelog**: https://github.com/oasisprotocol/cli/compare/{{ .PreviousTag }}...{{ .Tag }}
+
+homebrew_casks:
+  - name: oasis-cli
+    binary: oasis
+    directory: Casks
+    homepage: "https://github.com/oasisprotocol/cli"
+    description: "Command‑line interface for the Oasis Network"
+    commit_author:
+      name: github-actions[bot]
+      email: github-actions[bot]@users.noreply.github.com
+    commit_msg_template: "chore(brew): update {{ .ProjectName }} to {{ .Tag }}"
```

Comment:
I'm not familiar with the `homebrew_casks` workflow. But making a commit directly to the main branch will not work due to the security policies. Why would you need to commit to the cli repo?

Reply:
created a fix to open PRs instead trying to commit directly. more info on goreleaser `homebrew_casks` https://goreleaser.com/customization/homebrew_casks/

we need to commit to the cli repo (now with a PR) to let goreleaser auto‑add the updated `Casks/oasis-cli.rb` with each tag so brew users can brew upgrade oasis-cli, without us maintaining a separate repo/tap

Reply:
Could this simply still go under the `cli` repo but to a separate branch? Similarly to how `gh-pages` branch is commonly used for the github pages.

Reply:
after searching a bunch, this doesn’t looks to be possible. Homebrew only clones the default branch ( origin/HEAD ) of a tap